samples/drivers/audio/dmic/src/main.c

```diff
@@ -108,7 +108,7 @@ int main(void)
 
 	cfg.channel.req_num_chan = 1;
 	cfg.channel.req_chan_map_lo =
-		dmic_build_channel_map(0, 0, PDM_CHAN_LEFT);
+		dmic_build_channel_map(0, 0, PDM_CHAN_RIGHT);
```

Comment:
this change seems unrelated to the XIAO BLE Sense work, so at a minimum it should be a different commit. It looks like a more robust change would be to do LEFT, then RIGHT, then LEFT+RIGHT?

Reply:
Yeah I agree.